analyzer.go

```diff
@@ -86,9 +86,13 @@ func (wa *wslAnalyzer) run(pass *analysis.Pass) (interface{}, error) {
 			continue
 		}
 
-		// if the file is related to cgo the filename of the unadjusted position is a not a '.go' file.
 		fn := pass.Fset.PositionFor(file.Pos(), false).Filename
 
+		// if the file is related to cgo the filename of the unadjusted position is a not a '.go' file.
+		if !strings.HasSuffix(fn, ".go") {
+			continue
+		}
```

Comment:
Maybe we can just return `pass.Fset.PositionFor(file.Pos(), false).Filename` from `getFilename`, or even drop `getFilename` and just do this once? I don't think there's any need for first calling `getFilename` to get potential adjusted position/file and then again here to skip it.

Reply:
if we do that, the `line` directive will not work.